linters/jshint.py

```diff
@@ -0,0 +1,13 @@
+def options(ctx):
+    ctx.add_option('--jshint', action='store_true', help="Run JSHint on the JS files in the build")
+
+def configure(ctx):
+    if ctx.options.jshint:
+        try:
+            ctx.find_program('jshint', var='JSHINT')
+        except ctx.errors.ConfigurationError:
+            print "jshint was not found"
```

Comment:
You're swallowing the exception here. Also the print isn't adding anything beyond what `ctx.find_program(mandatory=False)` would print itself.